package.json

```diff
@@ -68,7 +68,8 @@
     "form-data": "0.1.4",
     "fs-extra": "0.30.0",
     "infinite-loop-loader": "1.0.1",
-    "jade": "1.11.0",
+    "jstransformer-clean-css": "^2.0.0",
+    "jstransformer-uglify-js": "^1.2.0",
```

Comment:
Jade has bultin transformers for uglify-js and clean-css. With Pug, they modularised all the tranformers and you need to install all the transformers you want to use separately. That's why I had to include them. Also, 'uglify-css' has been renamed to 'clean-css'.